src/Behavioral.Automation/Services/DriverService.cs

```diff
@@ -25,7 +25,6 @@ public sealed class DriverService : IDriverService
         public DriverService([NotNull] IScopeContextManager scopeContextManager)
         {
             _scopeContextManager = scopeContextManager;
-            Navigate("about:blank");
```

Comment:
I think this line was added intentionally by @SergeyRuzin, not sure if it should be removed.

Reply:
Don't worry I've confirmed with Sergey.

Reply:
I'm not sure it was added by me. Still we should check that it doesn't break anything

Reply:
I've checked. Working fine. The reason why I removed is that not possible to test desktop applications based on electronjs. The page becomes blank.